packages/www/docs/guides/upload-a-video/index.mdx

```diff
@@ -0,0 +1,682 @@
+---
+title: Upload a Video
+description: Livepeer's Upload a Video Guide
+metaTitle: Upload a Video Guide - Livepeer.com
+metaDescription: Livepeer's Upload a Video Guide
+---
+
+# Upload a Video
+
+This tutorial covers how to upload the video to Livepeer to allow viewers to
+playback stored video assets in your application.
+
+## Caveats
+
+- Have to manually check the status of the file using `List Task` method to see
```

Comment:
I still think this is a funny thing to ask a developer to do -- we should either tell readers that we're currently working on webhooks (which feels like the obvious thing that users are going to want) or give them instructions to do polling. @victorges thoughts on this?

If we decide that this okay, it needs to be rephrased: `Currently, users need to manually check the status of the file using List Task method (described below) to see if uploads have been completed. We're currently working on webhooks to make this easier for users.`

Reply:
Agreed! I don't think this is a caveat worth noting here as a big issue in the flow. It's just how things work for now while we don't have a more sophisticated interface. Webhooks will be nicer for communicating the state to a backend, but it won't make much of a difference for users building only a frontend (or simpler backends without custom state) anyway.

Also agree that we **must** provide instructions for polling along this guide. Can also link to the `video-nft` SDK (that needs rebranding) for an example on how to wait for a task to finish: https://github.com/livepeer/video-nft/blob/4e24bd46a884f7a0c23fe23b233cfd0c9a3bafad/src/minter.ts#L480

We can also create a simple `gist` or code snippet for that using axios.